activerecord/test/cases/insert_all_test.rb

```diff
@@ -3,6 +3,7 @@
 require "cases/helper"
 require "models/book"
 require "models/speedometer"
+require "models/measurement"
```

Comment:
:thinking: Maybe we can require this only in `test_upsert_all_works_with_partitioned_indexes` after the skip part since it will be used only there and for all other cases it will get loaded model having no real db table (since it is created conditionally).

Reply:
That's a nice catch! And I think is a good idea, since there's only one test in the whole file making use of it.

BTW I tested the `WHERE i.relkind IN ('i', 'I')` changes in PostgreSQL 9.4, 9.6 and 10 (plus 11) and the test suite passes without problems.